src/components/Button.jsx

```diff
@@ -2,7 +2,8 @@ import React from 'react';
 import PropTypes from 'prop-types';
 import styles from './Button.module.css';
 
-const Button = props => (
+/* eslint-disable react/require-default-props */
```

Comment:
Default props are there to ensure that the component has the information it needs and can prevent the component from breaking later. Just because the need my not be immediately apparent right now, doesn't mean that we shouldn't protect our code from failure. Don't let 'good enough' keep things from being great.

Appropriate DefaultProps in this case would be `null` for the `url` and `false` for disabled.